canonical/ubuntu.com
# Subcentre design update

## Done

- Updates subscription centre based on agreed upon changes found in [this prs conversation](https://github.com/canonical/ubuntu.com/pull/12345). You can see them in [this figma board](https://www.figma.com/file/XtTjD7N3QqQq2WGjUp0qDq/Subscription-Centre?node-id=202%3A469&t=JZtMO7s9lHCfvXBc-0)

Here are the following changes:
Bottom of the page: can the 2 buttons go above the fold? Now you don't really see them when landing on the page. Maybe removing the whitespace

Can we add the text about the privacy policy that's in the current subscription center near the "update prefs" button? Text: In submitting this form, I confirm that I have read and agree to [Canonical’s Privacy Notice](https://ubuntu.com/legal/data-privacy/newsletter) and [Privacy Policy](https://ubuntu.com/legal/data-privacy).

## QA

- Go to with [this extension](https://pastebin.canonical.com/p/pdnNYmbXPP/) and check the design changes have been applied

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-1325
